Formula/root.rb

```diff
@@ -52,10 +48,19 @@ class Root < Formula
   depends_on "xz" # for LZMA
   depends_on "zstd"
 
+  uses_from_macos "libxml2"
+
   conflicts_with "glew", because: "root ships its own copy of glew"
 
   skip_clean "bin"
 
+  # Can be removed post 6.22.08
+  patch :p0 do
+    url "https://github.com/root-project/root/commit/d113c9fcf7e1d88c573717c676aa4b97f1db2ea2.patch?full_index=1"
+    sha256 "6d0fd5ccd92fbb27a949ea40ed4fd60a5e112a418d124ca99f05f70c9df31cda"
+  end
+
+
```

Comment:
`brew audit` won't like the extra whitespace here...

Reply:
Yes, I forgot to run it again.

I could try building this on an M1 if there are any further issues, but really hoping this one patch will fix things.

Reply:
I guess we'll find out soon!

Reply:
Weird, the patch didn't apply, even though the file name is the same...

Reply:
Maybe the strip level is off?